dashboards/media-services-insights/msih.yaml

```diff
@@ -0,0 +1,816 @@
+            , ROW (62, ',NA,', ',', 'Remove any NA (need to do twice until regex can be tuned)', 'AWSElementalMediaConvert')
+            , ROW (90, '(,+)', ',', 'Remove multiple commas', 'AWSElementalMediaConvert')
+            , ROW (91, '^,', '', 'Remove leading comma', 'AWSElementalMediaConvert')
+            , ROW (92, ',$', '', 'Remove trailing comma', 'AWSElementalMediaConvert')
+            , ROW (99, '^(.*)$', '\{$1\}', 'Wrap in braces as json', 'AWSElementalMediaConvert')
+            , ROW (10, '-', ',', 'Prepare by replacing all hyphens with commas', 'AWSElementalMediaLive')
+            , ROW (11, '$', ',', 'Ensure last char is comma to avoid | rules; cleaned in the end', 'AWSElementalMediaLive')
+            , ROW (11, '^', ',', 'Ensure last char is comma to avoid | rules; cleaned in the end', 'AWSElementalMediaLive')
+            , ROW (20, '^,(.*?,).*?,(.*?$)', ',$1$2', 'Remove region', 'AWSElementalMediaLive')
+            , ROW (21, '^,MUX,', ',"IsStatmux":"Yes",', 'Handle statmux', 'AWSElementalMediaLive')
+            , ROW (22, '^,((.*?,){2})(.*?),', ',$1"Codec":"$3",', '', 'AWSElementalMediaLive')
+            , ROW (22, '^,((.*?,){3})(.*?),', ',$1"Resolution":"$3",', '', 'AWSElementalMediaLive')
+            , ROW (30, ',EML1,', ',"Pipeline":"Single",', '', 'AWSElementalMediaLive')
+            , ROW (30, ',EML2,', ',"Pipeline":"Standard",', '', 'AWSElementalMediaLive')
+            , ROW (30, ',IN,', ',"FlowDirection":"In",', '', 'AWSElementalMediaLive')
+            , ROW (30, ',OUT,', ',"FlowDirection":"Out",', '', 'AWSElementalMediaLive')
+            , ROW (30, ',L([0-9]*?),', ',"BitRate":"$1 mbps",', '', 'AWSElementalMediaLive')
+            , ROW (30, ',([0-9]*?),', ',"FrameRate":"$1 fps",', 'Should be the only digit-only entry', 'AWSElementalMediaLive')
+            , ROW (30, ',S,$', ',"Quality":"Standard",', '', 'AWSElementalMediaLive')
+            , ROW (30, ',E,$', ',"Quality":"Enhanced",', '', 'AWSElementalMediaLive')
+            , ROW (40, '"Codec":"IDLE"', ',"IsIdle":"Yes",', '', 'AWSElementalMediaLive')
+            , ROW (40, '"AUDIO"', '"Audio-only"', '', 'AWSElementalMediaLive')
+            , ROW (90, '(,+)', ',', 'Remove multiple commas', 'AWSElementalMediaLive')
+            , ROW (91, '^,', '', 'Remove leading comma', 'AWSElementalMediaLive')
+            , ROW (92, ',$', '', 'Remove trailing comma', 'AWSElementalMediaLive')
+            , ROW (99, '^(.*)$', '\{$1\}', 'Wrap in braces as json', 'AWSElementalMediaLive')
+         )  t ("priority", "pattern_exp", "replace_exp", "comment", "elemental_service")
+      )
+      , "emc_usage_translation_compiled" AS (
+         SELECT
+           "elemental_service"
+         , array_agg("pattern_exp" ORDER BY priority ASC) "patterns"
+         , array_agg("replace_exp" ORDER BY priority ASC) "reps"
+         FROM
+           "emc_usage_translation_rules"
+         GROUP BY 1
+      )
+      , "cur_pre_processing" AS (
+         SELECT
+           *
```

Comment:
If CUR will get additional columns such as SCAD or others this Athena query may fail with error the view is stale. Any reasons why not to list columns explicitly there from which used in the resulting select?